Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Thomas Breuer <sam@math.rwth-aachen.de>", "Sebastian Gutsche <gutsch
 version = "0.2.6"
 
 [deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
If we only need this for tests, we should instead add it to the `[extras]` section, shouldn't we?